docker/scout/scripts/main.py

```diff
@@ -227,6 +241,46 @@ def update_sett_gauge(sett_gauge, sett, sett_vaults, treasury_tokens):
         log.debug(e)
 
 
+def update_badger_sett_data_gauge(ibBTC_composition_gauge, badger_sett_lp):
+    name_sett_lp = badger_sett_lp.name_sett_lp
+    lp_address = badger_sett_lp.lp_address
+    name_peak = badger_sett_lp.name_peak
+    peak_address = badger_sett_lp.peak_address
+
+    badger_sett_lp_info = badger_sett_lp.describe()
+
+    log.info(
+        f"Processing ibBTC composition for Peak [bold]{name_peak}[/], Sett [bold]{name_sett_lp}"
+    )
+
+    for param, value in badger_sett_lp_info.items():
```

Comment:
Life is soooo much easier in grafana if you can include USD and BTC balances whenever possible.

The script already has all the data and stores it in vars as it moves through the loop.  So you should be able to very easily add a btc balance and usd balance param to each of the lps in the peaks, and yearn.  Look at how this is done in the lptokens and the setts (both of which have a usd value parameter)

Reply:
Added the DefiDollar fee wallet to "badger_wallets" so its ibBTC balance should get updated along with the Badger dev multisig wallet, which is representative of the fees received.

But can't calculate total fees paid out this way if either of the wallets move any ibBTC elsewhere.

Reply:
Added USD balances for each of the underlying Setts in the Peaks using the previously calculated Sett USD `pricePerShare`.